src/pages/proposal.tsx

```diff
@@ -57,59 +65,112 @@ import './proposals.css'
 
 // TODO: Review why proposals.css is being imported and use only proposal.css
 
+export type SelectedChoice = { choice?: string; choiceIndex?: number }
```

Comment:
Can you move this to `Proposal/types.ts`? or maybe `Votes/types.ts`? Also it may be useful to rename it to `SelectedVoteChoice`. A topic feedback can be a choice in this context too 🤔